common/config/rush/experiments.json

```diff
@@ -0,0 +1,28 @@
+/**
+ * This configuration file allows repo maintainers to enable and disable experimental
+ * Rush features. For full documentation, please see https://rushjs.io
+ */
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/experiments.schema.json",
+
+  /**
+   * Rush 5.14.0 improved incremental builds to ignore spurious changes in the pnpm-lock.json file.
+   * This optimization is enabled by default. If you encounter a problem where "rush build" is neglecting
+   * to build some projects, please open a GitHub issue. As a workaround you can uncomment this line
+   * to temporarily restore the old behavior where everything must be rebuilt whenever pnpm-lock.json
+   * is modified.
+   */
+  // "legacyIncrementalBuildDependencyDetection": true,
+
+  /**
+   * By default, rush passes --no-prefer-frozen-lockfile to 'pnpm install'.
+   * Set this option to true to pass '--frozen-lockfile' instead.
+   */
+  // "usePnpmFrozenLockfileForRushInstall": true,
+
+  /**
+   * If true, the chmod field in temporary project tar headers will not be normalized.
+   * This normalization can help ensure consistent tarball integrity across platforms.
+   */
+  // "noChmodFieldInTarHeaderNormalization": true
+}
```

Comment:
Codacy found an issue: [Unexpected character ('}' (code 125)): was expecting double-quote to start field name](https://app.codacy.com/gh/matteobruni/tsparticles/pullRequest?prid=6397531)